src/components/comments/comment-area-all-threads-thread.tsx

```diff
@@ -70,32 +62,37 @@ export function CommentAreaAllThreadsThread({
             )
           </div>
           <div>
-            {status === 'loading' ? (
-              <FaIcon icon={faSpinner} className="animate-spin-slow" />
-            ) : (
-              <select
-                className={clsx(
-                  'mb-1 rounded py-2 pl-4 pr-6 disabled:appearance-none disabled:py-1 disabled:text-center',
-                  status === 'open' ? 'bg-yellow' : 'bg-brandgreen-500'
-                )}
-                disabled={!canSetStatus}
-                value={status}
-                onChange={(e) => {
-                  setStatus(e.target.value)
-                  submitEvent('thread_status_set')
-                  void fetch(
-                    `/api/frontend/thread-status/set/${thread.id}?label=${e.target.value}`
-                  )
-                }}
-              >
-                <option value="open" className="bg-white">
-                  offen
-                </option>
-                <option value="done" className="bg-white">
-                  erledigt
-                </option>
-              </select>
-            )}
+            <select
+              className={clsx(
+                'mb-1 rounded py-2 pl-4 pr-6 disabled:appearance-none disabled:py-1 disabled:text-center',
+                status === 'open'
+                  ? 'bg-yellow'
+                  : status === CommentStatus.Done
+                  ? 'bg-brandgreen-500'
+                  : 'bg-gray-200'
+              )}
+              disabled={!canSetStatus}
+              value={status}
+              onChange={(e) => {
+                setStatus(e.target.value)
+                submitEvent('thread_status_set')
+                onMutate()
+                void mutateStatus({
+                  id: [thread.id],
+                  status: e.target.value as CommentStatus,
+                })
+              }}
+            >
+              <option value="open" className="bg-white">
+                offen
```

Comment:
I guess no translations because the whole feature is still a work in progress?

Reply:
we only use this feature for the german language instance, so all of the UI is not translated